modules/module.js

```diff
@@ -0,0 +1,162 @@
+export * from './module.js';
```

Comment:
[OPTIONAL] It is preferable to relocate the modules folder to the src folder to keep your project well-structured.